java/jdbc-reactive-extensions-intro/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.oracle.jdbc.reactive</groupId>
+  <artifactId>jdbc-reactive-extensions-intro</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <name>jdbc-reactive-extensions-intro</name>
+  <description>Introduction to JDBC Reactive Extensions with the Oracle
+    Database 23c Free — Developer Release</description>
```

Comment:
Please change 23c to 23ai.

Reply:
And please remove " - Developer Release" as starting with 23.4, it no longer is a Developer Release.

Reply:
changed to make it release-agnostic as requested. @Kuassim please review. Thanks